"use server";
export async function startError() {
  throw new Error("에러 발생");
}
